.github/ISSUE_TEMPLATE/implement-new-feature.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Implement new feature
+about: Create a new featrue
```

Comment:
How about:
Create an article about the motivation and specification of a new feature